src/java.base/share/classes/jdk/internal/foreign/CABI.java

```diff
@@ -85,6 +86,8 @@ private static CABI computeCurrent() {
             } else if (arch.equals("ppc64")) {
                 if (OperatingSystem.isAix()) {
                     return AIX_PPC_64;
+                } else if (OperatingSystem.isLinux()) {
+                    return LINUX_PPC_64;
                 }
```

Comment:
Please reorder these so the lines from upstream are unchanged:
```java
                if (OperatingSystem.isLinux()) {
                    return LINUX_PPC_64;
                } else if (OperatingSystem.isAix()) {
                    return AIX_PPC_64;
                }
```

Reply:
Re-ordered.